hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -254,6 +261,52 @@ public EvictionPolicyType getEvictionPolicyType() {
         }
     }
 
+    /**
+     * Returns the class name of the configured {@link EvictionPolicyComparator} implementation.
+     *
+     * @return the class name of the configured {@link EvictionPolicyComparator} implementation
+     */
+    @Override
+    public String getComparatorClassName() {
+        return comparatorClassName;
+    }
+
+    /**
+     * Sets the class name of the configured {@link EvictionPolicyComparator} implementation.
+     * <p/>
+     * Only one of the {@code comparator class name} and {@code comparator} can be configured in the eviction configuration.
+     *
+     * @param comparatorClassName the class name of the configured {@link EvictionPolicyComparator} implementation
+     * @return this EvictionConfig instance
+     */
+    public EvictionConfig setComparatorClassName(String comparatorClassName) {
```

Comment:
To be honest, I have to look up the code to see what this does. It's on my TODO. I think it's also not documented in the ref manual, at least not in the Near Cache chapter (maybe somewhere in the IMap or JCache eviction).